Project section (#1)
cheryllium/easy-html-css: _lessons/18-adding-some-color.md

Now, open the file in your browser, or refresh the page if you already had it open. Right-click anywhere on the background of the page, and select "Inspect". (In some browsers, the option may be called "Inspect Element" instead.)

This will open up a new panel which might open on the bottom or side of your browser. This panel is called the Inspector, and comes with several tools to poke around the code for your page!

The Inspector has a few different sections. In one section, you'll see the HTML code of your page, with an element highlighted. If it's not already highlighted, click on the `<body>` tag to highlight the body element. 

<div style="text-align: center">
  <img src="/assets/ch18_1.png" />
</div>

**Note** These screenshots were taken in Firefox, but other browsers have the same features; it will just look slightly different. I am also using dark mode; your inspector might be white instead. 

Now, either under it or to the side, you'll see a separate section that has all the CSS code that applies to the `body`. It will look something like this: 

<div style="text-align: center">
  <img src="/assets/ch18_2.png" />
</div>

> **Note** You might see some CSS here that you didn't write. I have some in my screenshot. This code comes from browser extensions and can be safely ignored. 

Reviewer: Can you temporarily disable your extensions for the purpose of this screenshot? This is going to be a bunch of confusing noise to beginners.

> **Note** You might see some CSS here that you didn't write. I have some in my screenshot. This code comes from browser extensions and can be safely ignored. 

Reviewer: ```suggestion
**Note** You might see some CSS here that you didn't write. This extra CSS code (if you have it) comes from browser extensions and can be safely ignored. 
```

> The CSS you did write is there too, however! And not only can you see it here, but you can also edit it and see changes on the page immediately. Next to the `#ffffff`, there is a little white circle. Click on it to bring up a color picker. Try using it to choose a different color. As you click on different colors, the background color of the page will change in real time!

Reviewer: In Chrome, it's a white square lol. Maybe add a note about that.

> The CSS you did write is there too, however! And not only can you see it here, but you can also edit it and see changes on the page immediately. Next to the `#ffffff`, there is a little white circle or square. Click on it to bring up a color picker. Try using it to choose a different color. As you click on different colors, the background color of the page will change in real time!

Reviewer: ```suggestion
The CSS you did write is there! And not only can you see it here, but you can also edit it and see changes on the page immediately. Next to the `#ffffff`, there is a little white circle or square. Click on it to bring up a color picker. Try using it to choose a different color. As you click on different colors, the background color of the page will change in real time!
```

<div class="screenshot">
  <img src="/assets/ch18_3.png" />
</div>

These changes are only happening as a preview in your browser. Once you pick a color you like, you still need to edit the CSS in your actual file to use that color. However, the browser inspector is a much faster way of trying out colors than editing your file and refreshing every time!

Go ahead and use the Inspector to pick a color you like for the background. Then, copy the hexadecimal code into your HTML file. 

### Sprucing up the Page Sections

Next, let's add some backgrounds and borders to each page section using the inspector. This time, right-click on the header and choose Inspect Element. Make sure that `<div id="header">` is selected in the HTML panel. Now, take a look at the panel with the CSS. 

Find the section of CSS that starts with just `#header`, and then move your mouse right under the last rule (`text-align: center;` in my screenshot) and click. A small text box will appear where you can type a new CSS rule. 

<div style="text-align: center">
  <img src="/assets/ch18_4.png" />

> </div>

Reviewer: This UX is pretty confusing (and briefly confused me the first time I tried this). I would suggest having an animated GIF showing exactly where to click for this specific example, instead of a static image.
